tests/ui_tests/app/app.py

```diff
@@ -95,19 +93,24 @@ def get_element(el):
             return WDW(self.driver, timer).until(condition(el))
         return get_element(locator)
 
+    @allure.step('Wait for element displayed')
```

Comment:
Can we add information about what element we are waiting for?

Reply:
need separate task for adding one more value to locators if it's necessary